### ISSUE TITLE
How to print encrypted pdf file in Android?

### ISSUE THREAD

**Issue by pdfdroid** [barteksc/AndroidPdfViewer#510]
While i am trying to print i am getting the error: **cannot print a password protected PDF file**

**Comment by barteksc** [comment]
This library does not support printing

**Comment by pdfdroid** [comment]
Thank you for your reply.
Any other way to handle this?

**Comment by 1stmetro** [comment]
Yes print the pdf off as a bitmap which is basically what it is, load it to memory and then dump it,
